wire-ios-sync-engine/Tests/Source/SessionManager/SessionManagerTests+EncryptionAtRestMigration.swift

```diff
@@ -0,0 +1,103 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import LocalAuthentication
+@testable import WireSyncEngine
+
+final class SessionManagerTests_EncryptionAtRestMigration: IntegrationTest {
```

Comment:
Can we avoid having _ on a class name? I think it's fine for **individual tests** but specifically for **classes** it will trigger some warnings as soon as we enable a specific **SwiftLint** rule that we currently have disabled.